core/docs/spark-qualification-tool.md

```diff
@@ -278,6 +278,7 @@ The tree structure of the output directory `${OUTPUT_FOLDER}/rapids_4_spark_qual
     ├── rapids_4_spark_qualification_output_execs.csv
     ├── rapids_4_spark_qualification_output_stages.csv
     ├── rapids_4_spark_qualification_output_mlfunctions.csv
+    ├── rapids_4_spark_qualification_output_mlfunctionsstageduration.csv
```

Comment:
Thanks for the feedback. I didn't see any other files having `_` so thought that was the convention. I will change it to `mlfunctions_totalduration` so that it implies total duration of ML functions excluding non-ML expressions/functions for a given app.